dd-sdk-android-sqldelight/README.md

```diff
@@ -0,0 +1,66 @@
+# Datadog Integration for SQLDelight
+
+## Getting Started 
+
+To include the Datadog integration for [SQLDelight][1] in your project, simply add the
+following to your application's `build.gradle` file.
+
+```
+repositories {
+    maven { url "https://dl.bintray.com/datadog/datadog-maven" }
+}
+
+dependencies {
+    implementation "com.datadoghq:dd-sdk-android:<latest-version>"
+    implementation "com.datadoghq:dd-sdk-android-sqldelight:<latest-version>"
+}
+```
+
+### Initial Setup
+
+Before you can use the SDK, you need to setup the library with your application
+context, your Client token and your Application ID.
+To generate a Client token and an Application ID please check **UX Monitoring > RUM Applications > New Application**
+in the Datadog dashboard.
+
+```kotlin
+class SampleApplication : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        val config = DatadogConfig.Builder("<CLIENT_TOKEN>", "<ENVIRONMENT_NAME>", "<APPLICATION_ID>").build()
+        Datadog.initialize(this, config)
+
+        GlobalRum.registerIfAbsent(RumMonitor.Builder().build())
+    }
+}
+```
+
+Following SQLDelight's [Generated API documentation][1], you just have to provide the `DatadogSqliteCallback` in the 
+`AndroidSqliteDriver` constructor.
+
+Doing this detects whenever a database is corrupted and sends a relevant
+RUM error event for it.
+
+Kotlin: 
```

Comment:
You probably don't need to say this :/ We don't say it anywhere else so it's a bit confusing.

Reply:
We are mentioning it also for Sqlite here : https://github.com/DataDog/dd-sdk-android/blob/master/docs/rum_collection.md. 
I really think we should tell the users what we are trying to do with these callbacks (this was actually corrected by the docs team, I am not sure what was the original phrase I added there).

Oh...I just realised that I added the [...] without the // there , will fix it now.